### ISSUE TITLE
Cant' set Date for FormPickerDateElement

### ISSUE THREAD

**Issue by darkwood-studios** [TheJuki/KFormMaster#77]
Hi there,

I just try to prefill a date picker form element with a Date object, but nothing happens. Is there anything wrong?

```
val date = DateUtil.getDateFromString("2015-10-10")
formBuilder.getFormElement<FormPickerDateElement>(1).dateValue = date
```

--- util ---
```
fun getDateFromString(dateString: String, format: String = "yyyy-MM-dd") : Date {
            val dateFormat = SimpleDateFormat(format, Locale.getDefault())
            var convertedDate = Date()
            try {
                convertedDate = dateFormat.parse(dateString)
            } catch (e : ParseException) {
                e.printStackTrace()
            }
            return convertedDate
        }
```

**Comment by TheJuki** [comment]
Are you using version 6.1.0? I fixed this kindof of issue.

**Comment by darkwood-studios** [comment]
yes. same issue with 6.1.0

**Comment by TheJuki** [comment]
Did you set the dateFormat?
formBuilder.getFormElement<FormPickerDateElement>(1).dateFormat = SimpleDateFormat("yyyy-MM-dd", Locale.US)

**Comment by darkwood-studios** [comment]
now my code looks like that:

```
val date = DateUtil.getDateFromString("2015-10-10")
        val dateFormElement = formBuilder.getFormElement<FormPickerDateElement>(1)
        dateFormElement.dateFormat = SimpleDateFormat("yyyy-MM-dd", Locale.getDefault())
        dateFormElement.dateValue = date
```

but nothing happens...

**Comment by TheJuki** [comment]
What do you mean by that? Does it not show up on the form? Does the date picker work?

**Comment by TheJuki** [comment]
Okay I found the issue. Working on a fix.

**Comment by TheJuki** [comment]
I just released [6.1.1](https://github.com/TheJuki/KFormMaster/releases/latest). Please update your dependency and let me know if setting the dateValue works for you.

**Comment by TheJuki** [comment]
I am closing this issue. If you have any issues feel free to reopen.
